src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball1 = Lottery.getRandomBall();
+        System.out.println(ball1);
+        Ball ball2 = Lottery.getRandomBall();
+        System.out.println(ball2);
+        Ball ball3 = Lottery.getRandomBall();
+        System.out.println(ball3);
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class)
**Use for loop for creating several objects of the same class**

Reply:
+ 
Used array of objects.

Reply:
+